src/components/common/AppInput.tsx

```diff
@@ -104,6 +123,30 @@ const AppInput = ({ value, confirmValue, setValue, placeholderText, contentType,
                     textContentType={contentType as any}
                     onFocus={handleFocus}
                     onBlur={handleBlur}
+                    editable={isEditable}
+                    focusable={isEditable}
+                />
+            </View>
+            {error && <Text style={styles.error}>{error}</Text>}
+        </View>
+    )
+
+    else if (contentType === ContentType.TEXT) return (
+        <View style={styleProps}>
+            <View>
+                <TextInput
+                    style={[styles.input, { borderColor }]}
+                    onChangeText={handleChange}
+                    value={value}
+                    placeholder={placeholderText}
+                    placeholderTextColor={Colors.BLUE400}
+                    secureTextEntry={false}
+                    textContentType={contentType as any}
```

Comment:
Use `as` only if you have to, try avoiding it here and if it takes more then 5 minutes leave it like this